source/lib/operators/not.ts

```diff
@@ -10,10 +10,15 @@ export const not = <T, P extends Predicate<T>>(predicate: P) => {
 	const originalAddValidator = predicate.addValidator;
 
 	predicate.addValidator = validator => {
-		const fn = validator.validator;
-		const message = validator.message;
+		const {validator: fn, message, invertedMessage} = validator;
+		const placeholder = Math.random().toString(16).slice(2);
```

Comment:
Can you move this to an util called `random-id`? https://github.com/sindresorhus/ow/tree/master/source/utils
```suggestion
		const placeholder = randomId();
```